src/pages/Preferences/Volumes/VolumeForm.js

```diff
@@ -0,0 +1,151 @@
+import React              from 'react';
+import deepClone          from 'mout/src/lang/deepClone';
+import style              from 'HPCCloudStyle/ItemEditor.mcss';
+
+// const types = {
```

Comment:
right now cumulus just takes the type 'ebs'

I tried working this in but commented it out for later.